service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DraftCMOControllerTest.java

```diff
@@ -56,8 +58,11 @@
     private ObjectMapper mapper;
 
     @Test
-    void aboutToStartCallbackShouldPopulateHearingDatesList() throws Exception {
-        Map<String, Object> data = ImmutableMap.of("hearingDetails", hearingDetails);
+    void aboutToStartCallbackShouldPopulateHearingDatesListAndSetAssigneeDropdownKeys() throws Exception {
```

Comment:
Please move it to operate test case. Test should do one thing and do it well. If method name contains words like 'and' it simple indication that test is checking too many things.

Reply:
I agree but it would seem there is conflicting opinions in the team!

Reply:
Conflicting and bad.

Reply:
What if we re-considered the test name `aboutToStartCallbackShouldPrepareCaseForCMO`

Reply:
That would be useful seeing as my story is going to be doing more of the same when I get round to editing that test